Doc/library/inspect.rst

```diff
@@ -1475,6 +1475,9 @@ generator to be determined easily.
 
    .. versionadded:: 3.5
 
+   .. versionchanged:: 3.11
+      Added requirement on ``cr_suspended`` attribute.
```

Comment:
Just wondering if we should add 'for coroutine-like object.', i.e.:

Added requirement on ``cr_suspended`` attribute for coroutine-like object.